### PR TITLE
[v4r0] backport SpaceOccupemcy and Downtime calendar

### PR DESCRIPTION
**Pull request by zmathe**
BEGINRELEASENOTES

NEW: Add SpaceOccupancy and Downtime calendar

ENDRELEASENOTES

### CODE REVIEW

File: WebApp/handler/DowntimesHandler.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
+###############################################################################
+# (c) Copyright 2019 CERN for the benefit of the LHCb Collaboration           #
+#                                                                             #
+# This software is distributed under the terms of the GNU General Public      #
+# Licence version 3 (GPL Version 3), copied verbatim in the file "LICENSE".   #
+#                                                                             #
+# In applying this licence, CERN does not waive the privileges and immunities #
+# granted to it by virtue of its status as an Intergovernmental Organization  #
+# or submit itself to any jurisdiction.                                       #
+###############################################################################
```
Comment (fstagni) [inline WebApp/handler/DowntimesHandler.py]: Remove the copyright CERN (also from the other files)

**Review by fstagni — COMMENTED** [review]
...I guess this also fixes the issue with SiteSummary page, right?

**Comment by zmathe** [comment]
No, it does not fix the issue of the SiteSummary. I though it is the server side. I will have a look that issue as well

**Review by fstagni — APPROVED** [review]

**Comment by zmathe** [comment]
@fstagni You can have a final review. I fixed all possible bugs and in addition I improved a bit the pages....

**Review by fstagni — APPROVED** [review]

**Pull request merged**